cm_sdk/client.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from cm_sdk.api import APIException
+
+class CloudManagerClient(object):
```

Comment:
It's kind of odd that this module is top-level but only for internal package use, while `cm_api` is in a package and for external use. You should pick one or the other, not both.